mcp/mcp-cli/src/main/java/software/amazon/smithy/java/mcp/cli/commands/ListBundles.java

```diff
@@ -29,13 +29,14 @@ public class ListBundles extends SmithyMcpCommand {
     protected void execute(ExecutionContext context) {
         var registry = context.registry();
         var installedBundles = context.config().getToolBundles().keySet();
-        for (BundleMetadata bundle : registry
-                .listMcpBundles()) {
+        for (Registry.RegistryEntry entry : registry.listMcpBundles()) {
+            var bundle = entry.getBundleMetadata();
             boolean isInstalled = installedBundles.contains(bundle.getName());
             var commandLine = spec.commandLine();
             System.out.println(commandLine
                     .getColorScheme()
-                    .string("@|bold " + bundle.getName() + (isInstalled ? " [installed]" : "") + "|@"));
+                    .string("@|bold " + bundle.getName() + (isInstalled ? " [installed]" : "") + "|@: "
+                            + entry.getTitle()));
```

Comment:
Can you add a println at the end of description. Otherwise descriptions are clobberred.

Reply:
What do you mean by "clobbered?" The output appears fine on my screen, if not a bit cramped.